lava/lib/dl/bootstrap/README.md

```diff
@@ -32,10 +34,12 @@ In Bootstrap training. An SNN is used to jumpstart an equivalent ANN model which
 
 ## Hybridization
 
+The dynamic estimation of ANN activation function may still not be enough to reduce the gap between SNN and it's equivalent ANN, especially when the inference timesteps are low and the networks grow deep. In such a scenario, one can look at a hybrid approach of directly training a part of the network as SNN layers/blocks while acclearating the rest of the layers/blocks with bootstrap training.
```

Comment:
... of THE ANN activation function...

Reply:
... between THE SNN and ITS equivalent ANN...

Reply:
.,.especially for few inference time steps and networks with increasing depth...